src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -625,13 +625,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
 
         Public Function IsMethodLevelMember(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsMethodLevelMember
```

Comment:
Boy this method was scarily broken:
EventStatements/PropertyStatements that *did* have blocks returned true, delegates, accessors, etc.